### ISSUE TITLE
Closed Captions Settings not respecting lang file

### ISSUE THREAD

**Issue by bigt11** [videojs/video.js#3968]
## Description
I noticed that the Closed Captions settings window only displays English.  I want to change it to Spanish so I decided to add it to the es.js lang file.  I was able to replace all the English words with he es.js file except for these specific words even though they are declared in the lang file.  The problem words are "Font Size", "Text Edge Style", "Font Family", "Done".  All the rest seem to translate.

## Steps to reproduce
1. Edit your es.js lang file
[es.js.txt](https://github.com/videojs/video.js/files/722402/es.js.txt)
2. Include your es.js file in your html
4. In your player settings change the language to "es"
5. Check the CC Settings

## Results
![cc_settings_not_respect_lang_file](https://cloud.githubusercontent.com/assets/3448998/22187951/14b3c79c-e0c4-11e6-8a01-ff24899d7216.jpg)

### Expected
The problem words are "Font Size", "Text Edge Style", "Font Family", "Done".  They are not swapping out to "es"

#### videojs
Video.js 5.14.1

**Comment by mister-ben** [comment]
@bigt11 Thanks for the report. We have a fix for this. I've also added in your strings to es.json, but we're still missing a few more translations for Spanish -- would you be up for updating that once this is merged in?

**Comment by bigt11** [comment]
I don't speak Spanish myself,  all I did was use the google English to Spanish translator, so don't rely on the Spanish translations 100%.

What are the other words that are missing?  If you have the additional words in a file I know someone who speaks Spanish that could probably help out, but don't quote me on that.

**Comment by bigt11** [comment]
also these additional CC Settings strings in the lang file in theory would apply to all languages.

**Comment by mister-ben** [comment]
OK. Google translate isn't great, so I've removed those stringsfrom the PR. Hopefully a native speaker can contribute a translation later instead.

**Comment by bigt11** [comment]
do you have the file that contains all the possible words that need to be translated?  The Spanish translation is something I need to get done for my project, so I can pass it on to you when I get it properly translated.

**Comment by mister-ben** [comment]
That would be great!

All translatable strings should be in lang/en.json as the English "template", and we update docs/translations-needed.md that lists out all the strings that are missing from each language file.

These are the versions not in master yet that include the strings from the captions dialog

https://github.com/mister-ben/video.js/blob/02fedd21f53ed01cead76572f32e56b3f0e146fe/lang/en.json
https://github.com/mister-ben/video.js/blob/02fedd21f53ed01cead76572f32e56b3f0e146fe/docs/translations-needed.md

**Comment by bigt11** [comment]
i am revisiting this, I checked out your language file here https://github.com/mister-ben/video.js/blob/02fedd21f53ed01cead76572f32e56b3f0e146fe/lang/en.json  and added them to my lang/en.js file, and did a google translate for my es.js version.  I am still getting the initial issues as stated in my first post of this issue.  For me the words "Font Size", "Text Edge Style", "Font Family", "Done", do not get swapped out for the Spanish versions.  I am on video.js Version 5.19.2.  I am not sure is there is a code issue preventing those words from being swapped out.

as a quick test, on the english en.js file I tried to change the above mentioned trouble words to something else, and it did not work either.
